README.md

```diff
@@ -1,2 +1,183 @@
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- INTRO -->
+# 📖 Hello Microverse <a name="about-project"></a>
+
+> Here my Hello Microverse page I created using HTML, CSS with Linters
+
+## 🛠 Built With <a name="built-with"></a>
+1. HTML
+2. CSS
+3. Linters
+### Tech Stack <a name="tech-stack"></a>
+
+> Tech Stack to be updated soon
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">Linters</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> - Added HTML
+> - Added CSS
+> - Added Linters
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+LIVE DEMO
+
+> Link to Live Demo will be added soon.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+### Prerequisites
+
+1. A Browser (Preferably Google Chrome)
+2. A Code Editor (Preferable Visual Studio)
+3. Internet Connection
+4. Git
+
+<!-- SETUP -->
+### Setup
+
+Clone this repository to your desired folder using:
+
+```sh
+  cd my-folder
+  git https://github.com/ishimwepati/Hello--Microverse.git
+```
+<!-- INSTALL -->
+
+### Install
+
+Install this project with the latest version of Git and a code editor.
+
+### Usage
+
+To run the project, execute the following command:
+> N/A
+### Deployment
+
+You can deploy this project using:
+>1.  A Code Editor
+>2. Latest version of Git
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Anyars Yussif**
+
+- GitHub: [@ishimwepati](https://https://github.com/ishimwepati)
+- Twitter: [@PatientMusafir1](https://twitter.com/PatientMusafir1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ishimwe-joseph-patient-0537b4155/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **Grades Management Portal**
+- [ ] **Fee Payment Portal**
+- [ ] **Student Assessment Portal**
+
```

Comment:
I suggest you remove this section if you don't have to work on some features.

Reply:
Yeah, i have removed them.